Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -473,17 +496,27 @@ bool PPCSymbolDB::SaveSymbolMap(const std::string& filename) const
   for (const auto& symbol : function_symbols)
   {
     // Write symbol address, size, virtual address, alignment, name
-    f.WriteString(fmt::format("{0:08x} {1:08x} {2:08x} {3} {4}\n", symbol->address, symbol->size,
-                              symbol->address, 0, symbol->name));
+    std::string line = fmt::format("  {0:08x} {1:06x} {2:08x} {3} {4}", symbol->address,
+                                   symbol->size, symbol->address, 0, symbol->name);
```

Comment:
As a note: the format for the `size` was changed from `{:08x}` to `{:06x}`. I don't think this change is harmful.

Reply:
I felt this would be a good change to make outputted maps be more in line with Codewarrior maps. If you think this should maybe be kept as some later PR or not at all, let me know.

Reply:
I think this change is fine and is unlikely to impact the symbol map loading/parsing.